.env.example

```diff
@@ -1,7 +1,6 @@
-DB_USER=
-DB_PASSWORD=
-DB_NAME=
-PORT=
-DB_TEST_USERNAME=
-DB_TEST_PASSWORD=
-DB_TEST_NAME=
\ No newline at end of file
+DB_USER=postgres
+DB_PASSWORD=bestmom
+DB_NAME=phantom
+DB_TEST_USERNAME=postgres
+DB_TEST_PASSWORD=bestmom
+DB_TEST_NAME=phantom
```

Comment:
Does this look like your db credentials ? 
refer to this
```
DB_USER='enter_your_db_username',
DB_PASSWORD=`enter_your_db_password'
```

Reply:
OMG:) Sure, I should change that

Reply:
I think this will be modified by @Cyebukayire  on his end this PR has nothing to do with them.